src/modules/commander/commander_params.c

```diff
@@ -620,3 +620,42 @@ PARAM_DEFINE_INT32(COM_ARM_MIS_REQ, 0);
  * @group Mission
  */
 PARAM_DEFINE_INT32(COM_POSCTL_NAVL, 0);
+
+/**
+ * Loss of position failsafe activation delay.
+ *
+ * This sets number of uSec that the position checks need to be failed before the failsafe will activate.
+ * The default value has been optimised for rotary wing applications. For fixed wing applications, a larger value between 5E6 and 10E6 should be used.
+ *
+ * @unit uSec
+ * @reboot_required true
+ * @group Commander
+ */
+PARAM_DEFINE_INT32(COM_POS_FS_DELAY, 1E6);
+
+/**
+ * Loss of position probation delay at takeoff.
+ *
+ * The probation delay is number of uSec that the EKF innovation checks need to pass for the positon to be declared good once it has been declared bad.
+ * The probation delay will be reset to this parameter value when takeoff is detected.
+ * After takeoff, if position checks are passing, the probation delay will reduce by one uSec for every lapsed uSec of valid position down to a minimum of 1E6 uSec.
+ * If position checks are failing, the probation delay will increase by COM_POS_FS_FACT micro seconds for every lapsed uSec up to a maximum of 100E6 uSec.
```

Comment:
Should `COM_POS_FS_FACT` be `COM_POS_FS_GAIN`?

Reply:
Will fix